launch-yocto.sh

```diff
@@ -109,8 +109,13 @@ launch_system_tests() {
 # Generate the test report and upload it
 generate_report() {
 
-  # Generate test report
+  # Replace test-report-pdf themes with SEAPATH one
   cd "${WORK_DIR}/ci/test-report-pdf"
+  rm -rf themes
+  mv ../seapath-themes themes
+  mv themes/theme.yml themes/sfl-theme.yml
```

Comment:
Is there a reason to keep here the sfl theme?

Reply:
The sfl theme is provided by test-report-pdf.
Here, i replace it by the seapath theme, but it is not clear. lets change these lines.